PolyEngine/Editor/Src/Widgets/Inspectors/ViewportInspectorWidget.cpp

```diff
@@ -209,7 +209,7 @@ void ViewportInspectorWidget::leaveEvent(QEvent* e)
 //------------------------------------------------------------------------------
 void ViewportInspectorWidget::resizeEvent(QResizeEvent* resizeEvent)
 {
-	if (gApp->EngineMgr->GetEngineState() == PolyEditor::eEngineState::NONE)
+	if (!Manager->GetEngine())
```

Comment:
This is very bad name for a field. Append some prefix, like `<sth>Manager`

Reply:
This will be fixed in next PR.